packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -66,7 +66,56 @@ function useDarkThemeBodyClassName( styles ) {
 	);
 }
 
-export default function EditorStyles( { styles } ) {
+function useParsedAssets( html ) {
+	return useMemo( () => {
+		const doc = document.implementation.createHTMLDocument( '' );
+		doc.body.innerHTML = html;
+		return Array.from( doc.body.children );
+	}, [ html ] );
+}
+
+function PrefixedStyle( { tagName, href, id, rel, media, textContent } ) {
+	const TagName = tagName.toLowerCase();
+
+	function onLoad( event ) {
+		const { sheet } = event.target;
+		const { cssRules } = sheet;
+
+		let index = 0;
+
+		for ( const rule of cssRules ) {
+			const { type, STYLE_RULE, selectorText, cssText } = rule;
+
+			if ( type !== STYLE_RULE ) {
+				continue;
+			}
+
+			if ( selectorText.includes( EDITOR_STYLES_SELECTOR ) ) {
+				return;
+			}
+
+			sheet.deleteRule( index );
+			sheet.insertRule(
+				'html :where(.editor-styles-wrapper) ' + cssText,
```

Comment:
Hey, I don't understand why we're prefixing the styles with `html :where(.editor-styles-wrapper)` instead of `.editor-styles-wapper`.

Reply:
If I'm not mistaken I think it's just to lower the specificity, but I'm also curious to know if that is indeed the case or not 🤔

Reply:
Yeah, I understand it lowers the specificity but I don't why. I guess my question is: why the styles loaded through a wp dependency should be any different than the styles loaded via `add_editor_styles`?

Reply:
I just used the same thing as #32659. But I guess if we consistently increase the specificity of all styles, this is not necessary. Cc @jasmussen.

Reply:
Where did we land in this conversation? I'd think we want to use the same we use for styles passed via `add_editor_style`?

Reply:
Adding the `:where` will ensure that the styles that are prefixed with .editor-styles-wrapper have the same specificity in the editor as they have in the frontend.